src/main/java/com/jvm_bloggers/frontend/newsletter_issue/NewsletterIssueDtoService.java

```diff
@@ -20,19 +20,24 @@
     private final NewsletterIssueDtoBuilder newsletterIssueDtoBuilder;
 
     public Optional<NewsletterIssueDto> getLatestIssue() {
-        Optional<NewsletterIssue> latestIssue =
-            newsletterIssueRepository.findFirstByOrderByPublishedDateDesc();
+        Optional<NewsletterIssue> latestIssue = newsletterIssueRepository
+            .findFirstByOrderByPublishedDateDesc();
         return latestIssue.map(newsletterIssueDtoBuilder::build);
     }
 
     public Optional<NewsletterIssueDto> findByIssueNumber(long issueNumber) {
-        Optional<NewsletterIssue> issue =
-            newsletterIssueRepository.findByIssueNumber(issueNumber);
+        Optional<NewsletterIssue> issue = newsletterIssueRepository.findByIssueNumber(issueNumber);
         return issue.map(newsletterIssueDtoBuilder::build);
     }
 
     public List<NewsletterIssueDto> findTop5ByOrderByPublishedDateDesc() {
         return newsletterIssueRepository.findTop5ByOrderByPublishedDateDesc()
             .stream().map(newsletterIssueDtoBuilder::build).collect(Collectors.toList());
     }
+
+    public List<NewsletterIssueDto> findAllByOrderByPublishedDateDesc() {
+        return newsletterIssueRepository.findAllByOrderByPublishedDateDesc().stream()
+            .map(newsletterIssueDtoBuilder::build).collect(Collectors.toList());
```

Comment:
@mszarlinski Ok, i will put each transformation on a stream from a new line. Why should i use static import to Collectors#toList?